src/tests/tile_creation.cpp

```diff
@@ -0,0 +1,437 @@
+/**
+ * @file tile_creation.cpp
+ * Unit tests for tile creation functions Make{Clear,Field,RoadTunnel,...}.
+ */
+#include "../stdafx.h"
+
+#include "../3rdparty/catch2/catch.hpp"
+
+#include "../map_func.h"
+#include "../clear_map.h"
+#include "../object_map.h"
+#include "../rail_map.h"
+#include "../road_map.h"
+#include "../station_map.h"
+#include "../tile_map.h"
+#include "../town_map.h"
+#include "../tree_map.h"
+#include "../tunnel_map.h"
+#include "../water_map.h"
+#include "../industry_map.h"
+#include "../depot_map.h"
+
+#include "../safeguards.h"
+
+
+/**
+ * Helper function to create a tile in a map for testing.
+ * All fields are initialized to ones, so the tests can verify that the Make* functions cleanly initialize all fields.
+ */
+static Tile taintedMockTile()
```

Comment:
We use Pascal case, never Dromedary case. 🙂 

```suggestion
static Tile TaintedMockTile()
```